src/renderer/components/interact/forms/Unbond.tsx

```diff
@@ -22,28 +21,60 @@ export const Unbond: React.FC<Props> = ({ onFinish: onFinishProp, isLoading = fa
   const [form] = Form.useForm<FormValues>()
 
   const onFinish = useCallback(
-    ({ memo }: FormValues) => {
+    ({ thorAddress, amount }: FormValues) => {
       onFinishProp({
-        memo
+        memo: getUnbondMemo(thorAddress, assetToBase(assetAmount(amount)))
       })
     },
     [onFinishProp]
   )
 
+  const amountValidator = useCallback(
+    (_, value: string, cb: (error?: string) => void) => {
+      const numberValue = Number(value)
+      if (numberValue <= 0 || Number.isNaN(numberValue)) {
+        cb(intl.formatMessage({ id: 'wallet.validations.graterThen' }, { value: 0 }))
+      }
+      if (Number(value) <= 0) {
+        cb(intl.formatMessage({ id: 'wallet.validations.graterThen' }, { value: 0 }))
+      } else {
+        cb()
+      }
+    },
```

Comment:
Why not using helpers from `helpers/form/validation` - see [validateTxAmountInput](https://github.com/thorchain/asgardex-electron/blob/1ca406c9fb60eab1a41008623faa81b8330f536d/src/renderer/components/wallet/txs/TxForm.util.ts#L18-L35)  as an example.